dask/dataframe/tests/test_shuffle.py

```diff
@@ -1132,3 +1132,18 @@ def test_set_index_nan_partition():
     d[d.a > 1].set_index("a", sorted=True)  # Set sorted index with 0 null partitions
     a = d[d.a > 3].set_index("a", sorted=True)  # Set sorted index with 1 null partition
     assert_eq(a, a)
+
+
+@pytest.mark.parametrize(
+    "npartitions",
+    [10, 1],
+)
+def test_sort_values(npartitions):
+    df = pd.DataFrame({"a": np.random.randint(0, 10, 100)})
+
+    ddf = dd.from_pandas(df, npartitions=npartitions)
+
+    tm.assert_frame_equal(
+        ddf.sort_values("a").compute().reset_index(drop=True),
+        df.sort_values("a").reset_index(drop=True),
+    )
```

Comment:
It's preferable to use `assert_eq` like the rest of the tests in this file do. That helper function checks properties of the dask dataframe before `compute` is called. By changing this test to use `assert_eq` you'll see the divisions issue that I mentioned above.

Reply:
Ah! I see - thanks!